front/src/applications/operationalStudies/components/ManageTrainSchedule/SubmitConfAddTrainSchedule.tsx

```diff
@@ -129,6 +129,14 @@ export default function SubmitConfAddTrainSchedule({ infraState, setIsWorking }:
         }
       }
     }
+    if (!osrdconf.simulationConf.pathfindingID) {
```

Comment:
To reach perfection haha

You can also move this test before the other one
```suggestion
    else if (!osrdconf.simulationConf.pathfindingID) {
```

Reply:
done :)